src/core/services/theming/theming.js

```diff
@@ -689,6 +758,40 @@ function generateAllThemes($injector, $mdTheming) {
     delete palette.contrastStrongLightColors;
     delete palette.contrastDarkColors;
 
+    function getContrastType(hueName) {
+      if (defaultContrast === 'light') {
+        if (darkColors.indexOf(hueName) > -1) return 'dark';
```

Comment:
Although the style guide doesn't specifically state it, we (well maybe just me) usually avoid one-line `if` statements as they can cause confusion if the formatter runs and changes the lines.

I know it's more lines of code, but can you use curly braces here and below and put the `return` statements on their own lines?